deploy/chart/templates/30_15-operatorgroup.crd.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: operatorgroups.operators.coreos.com
+spec:
+  group: operators.coreos.com
+  names:
```

Comment:
I believe you can get rid of `15-operatorgrouplist.crd.yaml` if you add `listKind: OperatorGroupList` here. Check out [how CSVs are handled in master](https://github.com/operator-framework/operator-lifecycle-manager/blob/05a838747633bb298e27ecf6209e25774ae0e034/deploy/chart/templates/30_02-clusterserviceversion.crd.yaml#L10).